examples/digital_credentials_api/signed_request_payload.json

```
@@ -0,0 +1,31 @@
+{
+  "client_id": "client.example.org",
```

Trivial, but federation entity_id's must be https urls:

```suggestion
  "client_id": "https://client.example.org/",
```

---

fixed..